.github/workflows/smoke-test.yml

```diff
@@ -3,18 +3,18 @@ on:
   workflow_dispatch:
     inputs:
       environment:
-        description: 'The environment to run tests against (qa, staging , production, productiondata or review)'
+        description: "The environment to run tests against (qa, staging , production, productiondata or review)"
```

Comment:
This formatting change is distracting. Could it be left out of this PR?

Reply:
Yes.